src/dds/rs-dds-device-proxy.cpp

```diff
@@ -174,25 +168,26 @@ dds_device_proxy::dds_device_proxy( std::shared_ptr< context > ctx, std::shared_
             auto const & default_profile = profiles[stream->default_profile_index()];
             for( auto & profile : profiles )
             {
+                LOG_DEBUG( "    " << profile->details_to_string() );
                 if( video_stream )
                 {
+                    auto video_profile = std::static_pointer_cast< realdds::dds_video_stream_profile >( profile );
                     auto added_stream_profile = sensor.add_video_stream(
```

Comment:
Why not call this `raw_stream_profile`? And the comment can apply to both motion and video

Reply:
OK, changing